packages/components/src/common/CloseButton/CloseButton.tsx

```diff
@@ -2,25 +2,30 @@ import clsx from "clsx";
 import React from "react";
 import Button from "../../Button";
 import CloseRoundedIcon from "../../Icons/CloseRounded";
-import { NotificationVariant } from "../Notifications/NotificationIcon";
 import styles from "./CloseButton.module.css";
 
+export type Variant = "brand" | "neutral" | "success" | "warning" | "alert";
+
 type CloseButtonProps = {
-  className?: string;
   /**
-   * The default color of the icon. Does not effect hover state.
+   * The color theme of the icon. Also affects the hover state.
    */
-  color: NotificationVariant;
-  onClose: () => void;
+  color: Variant;
   /**
-   * Size of the icon. Does not button size. The button is larger than the
+   * Size of the icon. Does not affect actual button size. The button is larger than the
    * icon to ensure the hit box is large enough (for accessibility).
    *
    * The string should be some number + px, rem, em, vh, etc. Ex: 2rem.
    * Recommendation: use "EdsSizeLineHeight" tokens from
    * @chanzuckerberg/eds-tokens/json/variables.json
    */
-  size: string;
+  size?: string;
+  /**
+   * Custom aria-label. If undefined, "close" will be used.
+   */
+  "aria-label"?: string;
+  onClose: () => void;
+  className?: string;
```

Comment:
I moved these to the bottom because I want the docs pages to have the props people will most want to play around with at the top of the list.